### PR TITLE
Added Skip Filters

### PR DESCRIPTION
**Pull request by honeycomb-cheesecake**
Changes documented in CHANGELOG.md.

### CODE REVIEW

File: decision/package-config/npm/package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (daveyarwood) [inline decision/package-config/npm/package.json:1]: Silly question: what is this file for, vs. the top-level package.json?

**Review by daveyarwood — DISMISSED** [review]
Thanks for the work you've been putting into this. It's great to see our deployment setup evolve.

File: decision/package-config/npm/package.json (hunk 1 above)
Comment (honeycomb-cheesecake) [reply decision/package-config/npm/package.json:1]: The overall project is a typescript project that is used to create the yaml spec. The file you are querying is the package.json for the output package that is saved to an npm registry. They are in effect different projects, and you can think of this file as a template file for the package which is used by the publish workflow. The publish workflow copies over this template, updates the version to the tag given for the github release.

Comment (honeycomb-cheesecake) [reply decision/package-config/npm/package.json:1]: Not silly

Comment (honeycomb-cheesecake) [reply decision/package-config/npm/package.json:1]: That folder is intended for any output package files needed, e.g. the stub `README.md` file that is also included.

Comment (daveyarwood) [reply decision/package-config/npm/package.json:1]: Makes sense. Thanks for the explanation!

**Pull request merged**